front/rule.common.php

```diff
@@ -80,10 +80,12 @@
     $rulecollection->checkGlobal(UPDATE);
 
    // Current time
+    /** @var array|number $start */
     $start = explode(" ", microtime());
     $start = $start[0] + $start[1];
```

Comment:
This would produce the exact same value.

```suggestion
    $start = microtime(true);
```

```php
echo microtime(true); // 1724057188.502
echo microtime();     // 0.50199900 1724057188